src/shipit_static_analysis/shipit_static_analysis/clang/format.py

```diff
@@ -193,6 +200,7 @@ def as_markdown(self):
             mode=self.mode,
             line=self.line,
             nb_lines=self.nb_lines,
+            is_new=self.is_new and 'yes' or 'no',
```

Comment:
Nit: the inline `if` is more readable (`yes` if self.is_new else 'no')

Reply:
I disagree, but OK :)